tests/test_crash_reports.py

```diff
@@ -107,6 +107,39 @@ def test_that_top_crashers_reports_links_work(self, mozwebqa, product):
             top_crashers = csp.release_channels
 
     @pytest.mark.nondestructive
+    def test_top_crasher_reports_tab_has_uuid_report(self, mozwebqa):
```

Comment:
When I run this test locally, against staging, it fails for me, but it passes against prod. Is it meant to also pass against staging? Here is the stack trace, and I have also attached the screen shot for the site when the failure occurs

![screenshot](https://cloud.githubusercontent.com/assets/142755/5751624/74e79c46-9c34-11e4-845d-c9dac10d315e.png)


```
================================================================= FAILURES ==================================================================
_______________________________________ TestCrashReports.test_top_crasher_reports_tab_has_uuid_report _______________________________________

self = <tests.test_crash_reports.TestCrashReports instance at 0x1095abe60>
mozwebqa = <pytest_mozwebqa.pytest_mozwebqa.TestSetup instance at 0x1095b0170>

    @pytest.mark.nondestructive
    def test_top_crasher_reports_tab_has_uuid_report(self, mozwebqa):
        csp = CrashStatsHomePage(mozwebqa)
        top_crashers = csp.click_last_product_top_crashers_link()
        crash_signature = top_crashers.click_first_signature()
        crash_signature.click_reports_tab()
        reports_table_count = len(crash_signature.reports)

        # verify crash reports table is populated
        Assert.greater(crash_signature.results_count_total, 0)
        Assert.greater(reports_table_count, 0, "No reports found")

        most_recent_report = crash_signature.reports[0]
        uuid_report = most_recent_report.click_report_date()

        # verify the uuid report page
>       Assert.not_equal(uuid_report.uuid_in_body, "", "UUID not found in body")

tests/test_crash_reports.py:125:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <pages.uuid_report.UUIDReport object at 0x10be53390>

    @property
    def uuid_in_body(self):
>       return self.selenium.find_element(*self._body_uuid_locator).text

pages/uuid_report.py:26:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <selenium.webdriver.firefox.webdriver.WebDriver object at 0x10be3a890>, by = 'css selector'
value = '#mainbody #report-header-details span:nth-of-type(1)'

    def find_element(self, by=By.ID, value=None):
        """
            'Private' method used by the find_element_by_* methods.

            :Usage:
                Use the corresponding find_element_by_* instead of this.

            :rtype: WebElement
            """
        if not By.is_valid(by) or not isinstance(value, str):
            raise InvalidSelectorException("Invalid locator values passed in")

        return self.execute(Command.FIND_ELEMENT,
>                            {'using': by, 'value': value})['value']

    def find_elements(self, by=By.ID, value=None):

../../.virtualenvs/Socorro-Tests/lib/python2.7/site-packages/selenium/webdriver/remote/webdriver.py:662:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <selenium.webdriver.firefox.webdriver.WebDriver object at 0x10be3a890>, driver_command = 'findElement'
params = {'sessionId': u'45782e2c-4165-8e47-ae90-f03bf3b1180e', 'using': 'css selector', 'value': '#mainbody #report-header-details span:nth-of-type(1)'}

    def execute(self, driver_command, params=None):
        """
            Sends a command to be executed by a command.CommandExecutor.

            :Args:
             - driver_command: The name of the command to execute as a string.
             - params: A dictionary of named parameters to send with the command.

            :Returns:
              The command's JSON response loaded into a dictionary object.
            """
        if self.session_id is not None:
            if not params:
                params = {'sessionId': self.session_id}
            elif 'sessionId' not in params:
                params['sessionId'] = self.session_id

        params = self._wrap_value(params)
        response = self.command_executor.execute(driver_command, params)
        if response:
>           self.error_handler.check_response(response)

../../.virtualenvs/Socorro-Tests/lib/python2.7/site-packages/selenium/webdriver/remote/webdriver.py:173:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x10be3abd0>
response = {u'name': u'findElement', u'sessionId': u'45782e2c-4165-8e47-ae90-f03bf3b1180e', u'status': 7, u'value': {u'message': ...mponents/driver-component.js:548', u'lineNumber': 5, u'methodName': u'fxdriver.Timer.prototype.setTimeout/<.notify'}]}}

    def check_response(self, response):
        """
            Checks that a JSON response from the WebDriver does not have an error.

            :Args:
             - response - The JSON response from the WebDriver server as a dictionary
               object.

            :Raises: If the response contains an error message.
            """
        status = response['status']
        if status == ErrorCode.SUCCESS:
            return
        exception_class = ErrorInResponseException
        if status in ErrorCode.NO_SUCH_ELEMENT:
            exception_class = NoSuchElementException
        elif status in ErrorCode.NO_SUCH_FRAME:
            exception_class = NoSuchFrameException
        elif status in ErrorCode.NO_SUCH_WINDOW:
            exception_class = NoSuchWindowException
        elif status in ErrorCode.STALE_ELEMENT_REFERENCE:
            exception_class = StaleElementReferenceException
        elif status in ErrorCode.ELEMENT_NOT_VISIBLE:
            exception_class = ElementNotVisibleException
        elif status in ErrorCode.INVALID_ELEMENT_STATE:
            exception_class = InvalidElementStateException
        elif status in ErrorCode.INVALID_SELECTOR \
                or status in ErrorCode.INVALID_XPATH_SELECTOR \
                or status in ErrorCode.INVALID_XPATH_SELECTOR_RETURN_TYPER:
            exception_class = InvalidSelectorException
        elif status in ErrorCode.ELEMENT_IS_NOT_SELECTABLE:
            exception_class = ElementNotSelectableException
        elif status in ErrorCode.INVALID_COOKIE_DOMAIN:
            exception_class = WebDriverException
        elif status in ErrorCode.UNABLE_TO_SET_COOKIE:
            exception_class = WebDriverException
        elif status in ErrorCode.TIMEOUT:
            exception_class = TimeoutException
        elif status in ErrorCode.SCRIPT_TIMEOUT:
            exception_class = TimeoutException
        elif status in ErrorCode.UNKNOWN_ERROR:
            exception_class = WebDriverException
        elif status in ErrorCode.UNEXPECTED_ALERT_OPEN:
            exception_class = UnexpectedAlertPresentException
        elif status in ErrorCode.NO_ALERT_OPEN:
            exception_class = NoAlertPresentException
        elif status in ErrorCode.IME_NOT_AVAILABLE:
            exception_class = ImeNotAvailableException
        elif status in ErrorCode.IME_ENGINE_ACTIVATION_FAILED:
            exception_class = ImeActivationFailedException
        elif status in ErrorCode.MOVE_TARGET_OUT_OF_BOUNDS:
            exception_class = MoveTargetOutOfBoundsException
        else:
            exception_class = WebDriverException
        value = response['value']
        if isinstance(value, basestring):
            if exception_class == ErrorInResponseException:
                raise exception_class(response, value)
            raise exception_class(value)
        message = ''
        if 'message' in value:
            message = value['message']

        screen = None
        if 'screen' in value:
            screen = value['screen']

        stacktrace = None
        if 'stackTrace' in value and value['stackTrace']:
            stacktrace = []
            try:
                for frame in value['stackTrace']:
                    line = self._value_or_default(frame, 'lineNumber', '')
                    file = self._value_or_default(frame, 'fileName', '<anonymous>')
                    if line:
                        file = "%s:%s" % (file, line)
                    meth = self._value_or_default(frame, 'methodName', '<anonymous>')
                    if 'className' in frame:
                        meth = "%s.%s" % (frame['className'], meth)
                    msg = "    at %s (%s)"
                    msg = msg % (meth, file)
                    stacktrace.append(msg)
            except TypeError:
                pass
        if exception_class == ErrorInResponseException:
            raise exception_class(response, message)
        elif exception_class == UnexpectedAlertPresentException and 'alert' in value:
            raise exception_class(message, screen, stacktrace, value['alert'].get('text'))
>       raise exception_class(message, screen, stacktrace)
E       NoSuchElementException: Message: Unable to locate element: {"method":"css selector","selector":"#mainbody #report-header-details span:nth-of-type(1)"}
E       Stacktrace:
E           at FirefoxDriver.prototype.findElementInternal_ (file:///var/folders/sk/1kj23lvn46n7pvr7_xk074wm0000gn/T/tmpUBhQpN/extensions/fxdriver@googlecode.com/components/driver-component.js:9641:26)
E           at fxdriver.Timer.prototype.setTimeout/<.notify (file:///var/folders/sk/1kj23lvn46n7pvr7_xk074wm0000gn/T/tmpUBhQpN/extensions/fxdriver@googlecode.com/components/driver-component.js:548:5)

../../.virtualenvs/Socorro-Tests/lib/python2.7/site-packages/selenium/webdriver/remote/errorhandler.py:166: NoSuchElementException
-------------------------------------------------------------- pytest-mozwebqa --------------------------------------------------------------
Failing URL: https://crash-stats.allizom.org/report/index/351b5270-6b0a-42cf-91fc-862c72150107
```